official/vision/beta/projects/yolo/dataloaders/classification_vision.py

```diff
@@ -0,0 +1,118 @@
+"""Classification parser."""
```

Comment:
Can you rename to classification _input.py?

Reply:
fixed